tests/unit/packages/test_normalize.py

```diff
@@ -297,6 +297,7 @@ def test_fix_pkg_config_trim_prefix_from_snap(
             f"{tmpdir}{fixed_prefix}"
         )
 
+    # pylint: disable=too-many-arguments
```

Comment:
Duplicate of line 317?

Reply:
Oh good catch I'll fix it in another PR real quick